examples/ophyd_async_db.json

```diff
@@ -0,0 +1,85 @@
+{
+    "uuid_filename_provider": {
+        "_id": "uuid_fp",
+        "active": true,
+        "args": [
+        ],
+        "creation": "Tue Feb 11 12:48:00 2025",
+        "device_class": "ophyd_async.core.UUIDFilenameProvider",
+        "documentation": null,
+        "kwargs": {
+        },
+        "last_edit": "Tue Feb 11 12:48:00 2025",
+        "name": "uuid_fp",
+        "type": "HappiItem"
+    },
+    "ymd_path_provider": {
```

Comment:
Small aside, I don't remember happi being able to generate json files where the top-level keys differ from the `name` field. Am I remembering wrong?

Reply:
To be honest these were mostly copy pasted and hand edited rather than generated via the client, so very likely this wouldn't be able to be generated in-code.